### PR TITLE
Minor optimization in the language detection code in ELF ##bin

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Use the boolean values set while iterating over the symbols to avoid checking twice for the same stuff

**Test plan**

Will make loading elfs with lots of symbols a bit faster (not much), but save some unnecessary loops

**Closing issues**
nope

### CODE REVIEW

**Comment by trufae** [comment]
@ret2libc it says &&, not ||, unless im missing something here, your comment is not correct. it will only stop if BOTH checks are true

**Comment by ret2libc** [comment]
rebase

**Comment by radare** [comment]
What's the need for rebasing? wasnt that something you were always against?

**Comment by ret2libc** [comment]
> What's the need for rebasing? wasnt that something you were always against?

Rebase/Merge from master. Do as you prefer. I'm not against rebasing. I've said it multiple times, but since we had always squashed+merged PRs, rebase only adds additional headache (in particular if one is not super familiar with git) because each commit can cause conflicts, while with a merge you just need to solve conflicts one time.

Let me rephrase it: pick udpates from master branch (either via rebase or via merge), as it seems some recent changes by @kazarmy caused the issue in appveyor.

**Comment by trufae** [comment]
we need at least 2 aprovals to merge, right?

**Comment by ret2libc** [comment]
> we need at least 2 aprovals to merge, right?

no, i already set the merge-when-green label 2 days ago, i was just waiting for ci
